src/senaite/patient/browser/patientfolder.py

```diff
@@ -154,12 +170,14 @@ def folderitem(self, obj, item, index):
         url = api.get_url(obj)
 
         # MRN
-        mrn = obj.getMRN() or obj.getId()
-        if mrn:
-            mrn = api.safe_unicode(mrn).encode("utf8")
-            item["mrn"] = mrn
-            item["replace"]["mrn"] = get_link(
-                url, value=mrn)
+        mrn = obj.getMRN()
+        if not mrn:
+            item["before"]["mrn"] = self.icon_tag("info", width=16)
```

Comment:
Just use the `get_image` function to get the info icon tag:
https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/utils/__init__.py#L709-L727

Reply:
Fixed with https://github.com/senaite/senaite.patient/pull/84/commits/3c6d98a03ee4b991db9d85a80d9eeea02d01cae1